PreMailer.Net/PreMailer.Net/PreMailer.Net.csproj

```diff
@@ -24,6 +24,7 @@
     <PackageTags>email css newsletter html</PackageTags>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>PreMailer.Net.snk</AssemblyOriginatorKeyFile>
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
```

Comment:
Prefer to keep this out, and rely on the CI/CD workflows to produce the package.